docs/content/latest/deploy/checklist.md

```diff
@@ -139,6 +139,18 @@ In practice, the clock drift would have to be orders of magnitude higher in orde
 
 {{< /note >}}
 
+## DNS caching
+
+If you are using name servers instead of static ips you may end up doing aggressive DNS lookups.
```

Comment:
"name servers" or "hostnames"

Reply:
@kmuthukk  Because @ddorian is referring to DNS caching, it is "name servers" and not hostnames. For DNS, "name servers" have a specific meaning (for example, see https://www.oreilly.com/library/view/linux-network-administrators/1565924002/ch06.html) Like you, I was thinking that hostname might be appropriate, but then did some research.

Reply:
Agree for "hostnames". The configuration points to a "hostname".